frontend/src/metabase/collections/components/ActionMenu/ActionMenu.tsx

```diff
@@ -101,7 +101,7 @@ function ActionMenu({
   const canArchive = canArchiveItem(item, collection);
   const canRestore = item.can_restore;
   const canDelete = canDeleteItem(item, collection);
-  const canCopy = canCopyItem(item);
+  const canCopy = onCopy && canCopyItem(item);
```

Comment:
Without this check, it shows the `Duplicate` dropdown menu item when we don't have an actual action for it

Reply:
I'd use `getIsEmbeddingSdk` selector for this. Otherwise, it's hard to understand this change.

Reply:
@heypoom what do you think? I personally think the change makes sense here - if we don't have an action for duplicating the item, we shouldn't show it in the menu. I know it's an SDK specific case but I think it also makes sense in general

Reply:
that's fair, I read somewhere down below you have a similar logic that checked for the passed prop I don't remember what is was about tho.

Reply:
I read it as "when the copy handler is provided and this item can be copied", so the logic already makes sense to me. I think what it does is clear, but the reason behind it might not be.

In this case, imo a small comment on top that indicates that this is currently used to support the embedding use case is good enough if we want to clarify.